packages/next-swc/crates/next-core/src/next_import_map.rs

```diff
@@ -1022,6 +1052,16 @@ fn export_value_to_import_mapping(
     }
 }
 
+fn insert_exact_alias_map(
+    import_map: &mut ImportMap,
+    project_path: Vc<FileSystemPath>,
+    map: IndexMap<&'static str, String>,
+) {
+    for (pattern, request) in map {
+        import_map.insert_exact_alias(pattern, request_to_import_mapping(project_path, &request));
```

Comment:
Nit: since we're just reffing the `request` string, can we use `&'static str` instead of `String`?

Reply:
This was kinda "forward looking" as I'm using `format!()` in https://github.com/vercel/next.js/pull/56675